masterlist.yaml

```
@@ -10058,7 +10068,7 @@ plugins:
       - *optional
       - *requiresBashOrSmash
     clean:
-      - crc: 0x1D19D916
+      - crc: 0x7C1ABB8D
```

```suggestion
      - crc: 0x1D19D916
```

This is the checksum of v2.4 of `Weapons Armor Clothing & Clutter Fixes.esp`, not `WACCF_BashedPatchLvlListFix.esp`.